.zenodo.json

```diff
@@ -451,12 +456,6 @@
       "name": "Durnez, Joke",
       "orcid": "0000-0001-9030-2202"
     },
-    {
-      "name": "Mertz, Fred"
-    },
```

Comment:
From earlier comment, this appears to be a psuedonym. Let me know if I should restore it.